src/libraries/System.Runtime.InteropServices/tests/System.Runtime.InteropServices.UnitTests/System/Runtime/InteropServices/NativeMemoryTests.cs

```diff
@@ -435,5 +435,108 @@ public void ReallocSmallerToLargerTest()
 
             NativeMemory.Free(newPtr);
         }
+
+        [Theory]
+        [InlineData(1, 0)]
```

Comment:
Unsure where, but we should also add the non-null, but `0` size case for `ZeroMemory`.

Reply:
Sure! I've added some tests for this in 868b9f416c09cc2248fc73adfbc025425ea7dbab 🙂